src/services/gitlab/client.js

```diff
@@ -148,4 +148,10 @@ export default class Client {
       .post(`/api/v4/projects/${project_id}/merge_requests/${iid}/todo`)
       .then(result => result.data);
   }
+
+  fetchProjects() {
+    return this.client
+      .get(`/api/v4/projects?membership=true`)
```

Comment:
Maybe default to `starred` projects first and order them by `name`

Reply:
Do you mean to only list starred projects? or to first order by starred projects and then by name?

Reply:
Display only starred projects and order them by name. Parameters are available in the API